vmm/src/api/openapi/cloud-hypervisor.yaml

```diff
@@ -380,6 +380,12 @@ components:
         hotplug_method:
           type: string
           default: "acpi"
+        shared:
+          type: boolean
+          default: false
+        hugepages:
+          type: boolean
+          default: false
```

Comment:
Could we also remove the `file` property?

Reply:
My recommendation is to keep it and mark it as deprecated and remove after 0.7.0 is released (we can open an issue for that.)

Reply:
@rbradford @sameo I just opened the issue https://github.com/cloud-hypervisor/cloud-hypervisor/issues/1063, and assigned it to myself. I will clean it up after release v0.7.0.